Command/RequestPasswordResetHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Command;
+
+use SumoCoders\FrameworkMultiUserBundle\Event\PasswordResetTokenCreated;
+use SumoCoders\FrameworkMultiUserBundle\User\PasswordReset as UserPasswordReset;
+use SumoCoders\FrameworkMultiUserBundle\User\UserRepositoryCollection;
+use Swift_Mailer;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class RequestPasswordResetHandler
+{
+    /**
+     * @var UserRepositoryCollection
+     */
+    private $userRepositoryCollection;
+
+    /**
+     * @var EventDispatcher
+     */
+    private $dispatcher;
+
+    /**
+     * PasswordResetRequestHandler constructor.
+     *
+     * @param UserRepositoryCollection $userRepositoryCollection
+     * @param EventDispatcherInterface $dispatcher
+     */
+    public function __construct(
+        UserRepositoryCollection $userRepositoryCollection,
+        EventDispatcherInterface $dispatcher
+    ) {
+        $this->userRepositoryCollection = $userRepositoryCollection;
+        $this->dispatcher = $dispatcher;
+    }
+
+    /**
+     * Creates a password reset token and sends an email to the user.
+     *
+     * @param RequestPasswordReset $command
+     *
+     * @return int
```

Comment:
SwiftMailer stuurt een in terug met het aantal mails, kan op deze manier ook getest worden